modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -16,7 +16,7 @@
             </div>
 
             <div class="apos-login__body" v-show="loaded">
-              <form>
+              <form ref="form" @submit="submit">
```

Comment:
You do not need a ref and a DOM event handler to preventDefault, see:

https://vuejs.org/v2/guide/events.html#Event-Modifiers

Reply:
Oh, yes. I forgot about this trick. Done.